Features/Context/SubContext/CommonActions.php

```diff
@@ -286,6 +286,18 @@ public function selectFromUniversalDiscovery($path)
     }
 
     /**
+     * @When I select the :path folder in the new Universal Discovery Widget
```

Comment:
It's not for you to fix, @micszo, but we need to name this thing. Nobody loves being called "the new XXX" :)

There was somthing about "Finder" but a) it might be copyrighted b) it includes explorer (the pane based thing), search and on-the-fly content creation, so it's probably not a good name.